backend/grant/app.py

```diff
@@ -1,19 +1,46 @@
 # -*- coding: utf-8 -*-
 """The app module, containing the app factory function."""
 import sentry_sdk
-from flask import Flask
+from animal_case import animalify
+from flask import Flask, Response, jsonify
 from flask_cors import CORS
 from flask_security import SQLAlchemyUserDatastore
 from flask_sslify import SSLify
+from sentry_sdk.integrations.flask import FlaskIntegration
+
 from grant import commands, proposal, user, comment, milestone, admin, email, blockchain, task, rfp
 from grant.extensions import bcrypt, migrate, db, ma, security
 from grant.settings import SENTRY_RELEASE, ENV
-from sentry_sdk.integrations.flask import FlaskIntegration
 from grant.utils.auth import AuthException, handle_auth_error, get_authed_user
 
 
+class JSONResponse(Response):
+    @classmethod
+    def force_type(cls, rv, environ=None):
+        if isinstance(rv, dict) or isinstance(rv, list) or isinstance(rv, tuple):
```

Comment:
`flask` handler return values will have to be `dict`s, `list`s or `tuple`s from now on. This seem right to me, but I might be missing something.